Add/change info on configuring Windows updates

heatherpoulsen opened the pull request:
Edited section formerly called "Disable automatic Windows Update on all nodes" to better describe default behaviors and recommendations.

heatherpoulsen committed:
Add/change info on configuring Windows updates

Edited section formerly called "Disable automatic Windows Update on all nodes" to better describe default behaviors and recommendations.

raunakpandya commented on articles/service-fabric/service-fabric-patch-orchestration-application.md:
```diff
@@ -130,9 +130,11 @@ To enable the repair manager service:
+### Configure updates to an administrator location on all nodes
 
-Automatic Windows updates might lead to availability loss because multiple cluster nodes can restart at the same time. The patch orchestration app, by default, tries to disable the automatic Windows Update on each cluster node. However, if the settings are managed by an administrator or group policy, we recommend setting the Windows Update policy to “Notify before Download” explicitly.
+All Windows 10 operating systems connected to the Internet will, by default, pull updates from the Microsoft Update infrastructure if not otherwise configured through administrative controls; this includes cluster nodes. If systems are not configured to appropriate controls, cluster nodes can have availability loss failures from multiple cluster nodes restarting simultaneously. The patch orchestr ...
```
Couple of comments:
1. The scenario is not restricted to Windows 10 BTW. We do support WS2012R2 and WS2016 clusters as well. 
2. I don't quite understand what do you mean by administrative controls/ location here? Can you explain? To be honest I am not sure if its clear for people what to do. 
3. What do you mean by with the last line? - enhance the Update stack for Azure ?

heatherpoulsen committed:
Update section on Windows Updates

Change title from "Disable automatic Windows Update on all nodes" to "Configure Windows Updates for all nodes" to more accurately describe that settings can be configured.

heatherpoulsen reviewed (commented):
Title update only.
